vehicle/mkv/software/brakelight_bspd/config.h

```diff
@@ -0,0 +1,51 @@
+#include "libs/adc/api.h"
```

Comment:
Nit/question, is worth renaming this file to bspd_config.h purely in an effort to change the line in the BUILD file to smthg more specific?

Reply:
Good idea!